themes/blue-owl.omp.json

```diff
@@ -0,0 +1,131 @@
+{
+  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+  "final_space": false,
+  "osc99": true,
+  "console_title": true,
+  "console_title_style": "template",
+  "console_title_template": "{{if .Root}} ⚡ {{end}}{{.Folder | replace \"~\" \"🏚\" }} @ {{.Host}}",
+  "blocks": [
+    {
+      "type": "prompt",
+      "alignment": "left",
+      "segments": [
+        {
+          "type": "root",
+          "style": "plain",
+          "foreground": "#FFEB3B",
+          "background": "#a313a8",
+          "properties": {
+            "root_icon": "⚡"
```

Comment:
Missed a few 😉

Reply:
That is an emoji, https://emojipedia.org/high-voltage/ any recommendation on how to proceed is much appreciated.

Reply:
Fixed the folder_icon to \u representation, hope that covers it all.

Reply:
Let's leave it!